style.css

```diff
@@ -0,0 +1,77 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+}
+
+body {
+    background: linear-gradient(135deg, #6e8efb, #a777e3);
```

Comment:
Please go and get more followers and work harder. Your code seems like chatgpt geerated code